docs/modules/hive/pages/usage-guide/security.adoc

```diff
@@ -0,0 +1,44 @@
+= Security
+
+== Authentication
+Currently, the only supported authentication mechanism is Kerberos, which is disabled by default.
+For Kerberos to work a Kerberos KDC is needed, which the users need to provide.
+The xref:home:secret-operator:secretclass.adoc#backend-kerberoskeytab[secret-operator documentation] states which kind of Kerberos servers are supported and how they can be configured.
+
+=== 1. Prepare Kerberos server
+To configure HDFS to use Kerberos you first need to collect information about your Kerberos server, e.g. hostname and port.
+Additionally, you need a service-user which the secret-operator uses to create principals for the HDFS services.
+
+=== 2. Create Kerberos SecretClass
+The next step is to enter all the necessary information into a SecretClass, as described in xref:home:secret-operator:secretclass.adoc#backend-kerberoskeytab[secret-operator documentation]. The following guide assumes you have named your SecretClass `kerberos`.
+
+=== 3. Configure HDFS to use SecretClass
+The next step is to configure your HdfsCluster to use the newly created SecretClass. Please follow the xref:hdfs:usage-guide/security.adoc[HDFS security guide] to set up and test this.
+Please make sure to use the SecretClass named `kerberos`. It is also necessary to configure 2 additional things in HDFS:
+
+* define group mappings for user permissions with `hadoop.user.group.static.mapping.overrides`
+* tell HDFS that Hive will be "passing through" users: i.e. Hive does not need any _direct_ access permissions for itself, but should be able to impersonate Hive users when accessing HDFS. This is done by setting `hadoop.proxyuser.hive.users` and `hadoop.proxyuser.hive.hosts`.
```

Comment:
```suggestion
* Define group mappings for users with `hadoop.user.group.static.mapping.overrides`
* Tell HDFS that Hive is allowed to impersonate other users, i.e. Hive does not need any _direct_ access permissions for itself, but should be able to impersonate Hive users when accessing HDFS. This can be done by e.g. setting `hadoop.proxyuser.hive.users=*` and `hadoop.proxyuser.hive.hosts=*` to allow the user `hive`´to impersonate all other users.
```